circuitpython_community_library_list.md

```diff
@@ -124,6 +124,7 @@ Here is a listing of current CircuitPython Community Libraries. These libraries
 * [CircuitPython MorseCode](https://github.com/jposada202020/CircuitPython_MorseCode.git) Circuitpython library to crete Morse code  ([PyPi](https://pypi.org/project/circuitpython-MorseCode)) \([Docs](https://circuitpython-morsecode.readthedocs.io/en/latest/))
 * [CircuitPython Noise](https://github.com/todbot/CircuitPython_Noise.git) ([PyPi](https://pypi.org/project/circuitpython-noise)) \([Docs](https://circuitpython-noise.readthedocs.io/en/latest/)) Simplex (Perlin-like) noise generation
 * [CircuitPython_OhmsLaw](https://github.com/CedarGroveStudios/CircuitPython_OhmsLaw.git) Calculate an Ohm’s Law result from two input parameters \([Docs](https://github.com/CedarGroveStudios/CircuitPython_OhmsLaw/blob/main/media/pseudo_rtd_cedargrove_ohmslaw.pdf))
+* [CircuitPython_Org_DispalyIO_Cartesian](https://github.com/circuitpython/CircuitPython_Org_DisplayIO_Cartesian.git) Graph cartesian widget for displayio ([PyPi](https://pypi.org/project/circuitpython-displayio-cartesian/) \([Docs](https://circuitpython-displayio-cartesian.readthedocs.io/en/latest/))
```

Comment:
```suggestion
* [CircuitPython_Org_DisplayIO_Cartesian](https://github.com/circuitpython/CircuitPython_Org_DisplayIO_Cartesian.git) Graph cartesian widget for displayio ([PyPi](https://pypi.org/project/circuitpython-displayio-cartesian/) \([Docs](https://circuitpython-displayio-cartesian.readthedocs.io/en/latest/))
```

Reply:
Thank you, fixed with the latest commit.